src/core/search/autocompleteresponsetransformer.js

```diff
@@ -57,8 +57,7 @@ export default class AutoCompleteResponseTransformer {
       filter: transformedFilter,
       key: result.key,
       matchedSubstrings: result.matchedSubstrings,
-      value: result.value,
-      shortValue: result.shortValue
```

Comment:
Removing this since the core's response no longer contains a `shortValue` property.